versions/1/integration/elasticsearch/spec.yml

```diff
@@ -29,8 +29,8 @@ spec:
           contentMediaType: "application/json"
           required: false
     - description: Folder containing Elasticsearch Transforms
-      # https://www.elastic.co/guide/en/elasticsearch/reference/current/transforms.html
       type: folder
       name: transform
       required: false
-      $ref: "./transform/spec.yml"
\ No newline at end of file
+      $ref: "./transform/spec.yml"
+      # https://www.elastic.co/guide/en/elasticsearch/reference/current/transforms.html
```

Comment:
Wow, didn't expect it to be a problem. Maybe we can put this comment inside the `transform/spec.yml` file?